server/services/ciagInductionService.test.ts

```diff
@@ -1,20 +1,13 @@
 import type { OtherQualifications, WorkAndInterests } from 'viewModels'
-import { HmppsAuthClient } from '../data'
 import CiagInductionClient from '../data/ciagInductionClient'
 import CiagInductionService from './ciagInductionService'
 
 describe('ciagInductionService', () => {
-  const hmppsAuthClient = {
-    getSystemClientToken: jest.fn(),
-  }
   const ciagClient = {
     getCiagInduction: jest.fn(),
   }
 
```

Comment:
I have removed `hmppsAuthClient` from the service class implementation, as it wasnt being used